internal/abci/abci.go

```diff
@@ -245,19 +246,38 @@ func NewAbciApp(ctx context.Context, cfg *AbciConfig, snapshotter SnapshotModule
 		// ABCI and applied below.
 	}
 
+	var migrationParams *common.MigrationContext
+	startHeight := app.consensusParams.Migration.StartHeight
+	endHeight := app.consensusParams.Migration.EndHeight
+
+	if startHeight != 0 && endHeight != 0 {
+		// set this only if "migrate_from" is configured
```

Comment:
What should be understood from this doc?  That the caller should have already ensured that migrate_from is set if these are non-zero?

Reply:
That's old comment, carried around. removing this